03_launch_mgmt_cluster.sh

```diff
@@ -480,31 +480,32 @@ if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
   kubectl create namespace metal3
 fi
 
-if [ "${CAPM3_VERSION}" != "v1alpha4" ]; then 
-  launch_baremetal_operator
-fi
-
 if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
   patch_clusterctl
   launch_cluster_api_provider_metal3
 fi
 
-launch_ironic
-
 if [ "${CAPM3_VERSION}" != "v1alpha4" ]; then 
-  apply_bm_hosts
+  launch_baremetal_operator
 fi
 
-if [ "${EPHEMERAL_CLUSTER}" != "tilt" ] && [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
+launch_ironic
+
+if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
+  if [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
+    BMO_NAMESPACE="capm3-system"
+    BMO_CAPM3_PREFIX="capm3-"
+  fi
```

Comment:
@ardaguclu hi, thanks for the PR. Why not to re-use already existing and BMO aware vars [here](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L171-L172) instead of re-defining the new ones here?